### PR TITLE
clarifications to release recipe

### PR DESCRIPTION
**Pull request by bkmartinjr**
Clarifications to the release recipe, based on my experience doing it for the first time.

Also ran it through prettier - so ignore whitespace changes.

### CODE REVIEW

File: docs/release_process.md [hunk 1]
```diff
@@ -14,35 +14,40 @@ The release process should result in the following side-effects:
...
+    - install required tools: `pip install -r requirements-dev.txt`
+    - Define the release version number, using [semantic versioning](https://semver.org/),
+      and specifying all three digits (eg, 0.3.0)
+    - Write the release title and release notes and add to
+      [release notes document](https://docs.google.com/document/d/1KnHwkYfhyWO5H8BDcMu7y3ogjvq5Yi4OwpmZ8DB6w0Y/edit)
 2.  Create a release branch, eg, `release-version`
 3.  In the release branch:
-    - Run `bumpversion --config-file .bumpversion.cfg [major | minor | patch]`
+    - Run `bumpversion --config-file .bumpversion.cfg [major | minor | patch]`,
+      were you choose major/minor/patch depending on which part of the version
```
Comment (csweaver) [inline docs/release_process.md]: were->where

File: docs/release_process.md [hunk 2]
```diff
@@ -14,35 +14,40 @@ The release process should result in the following side-effects:
...
+      is being dumped (eg, 0.2.9->0.3 is minor).
```
Comment (csweaver) [inline docs/release_process.md]: dumped->bumped

File: docs/release_process.md (hunk 1 above)
Comment (bkmartinjr) [reply docs/release_process.md]: blush

File: docs/release_process.md (hunk 2 above)
Comment (bkmartinjr) [reply docs/release_process.md]: sigh!

**Pull request merged**